pkg/models/monitoring/sort_page.go

```diff
@@ -50,9 +50,22 @@ func (w wrapper) Less(i, j int) bool {
 	p := w.MetricValues[i]
 	q := w.MetricValues[j]
 
-	if p.Sample.Value() == q.Sample.Value() {
+	// Place Nil to the tail.
+	if p.Sample == nil && q.Sample != nil {
+		return false
+	}
+	if p.Sample != nil && q.Sample == nil {
+		return true
+	}
+
+	// If both Samples are Nil or have the same metric value, sort by resource name
+	if p.Sample == q.Sample || p.Sample[1] == q.Sample[1] {
 		return p.Metadata[w.identifier] < q.Metadata[w.identifier]
```

Comment:
this looks like for ascending order only?
what about descending?

Reply:
The ranking rule is like this, because `p` and `q` has the same value (for example, cpu utilization for node `i-js38t` and node  `i-2p9s0` are both 35.6%), to prevent random ranking, sort `p` and `q` by their resource name in **ASCENDING** order. As a result, `i-2p9s0` take precedence over  `i-js38t`. 

I didn't change the ranking rule since ks 2.0. So if the equal value case happens, resource name sort order is always **ASCENDING**, and won't be affected by user-specified `order`.

Reply:
ok, that's fine to me